src/translations/screens/subscreens/SmartParkAndRide.ts

```diff
@@ -193,7 +192,7 @@ const SmartParkAndRideTexts = {
           'Les meir på atb.no',
         ),
       },
-      buttonText: _('Legg til kjøretøy', 'Add vehicle', 'Legg til køyretøy'),
+      buttonText: _('Hjelp og kontakt', 'Help and contact', 'Hjelp og kontakt'),
```

Comment:
Is this text change intended? I can't see it being used any differently than before 🤔